src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,101 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final User TOO_YOUNG = new User(16, "Bob", "passwd");
```

Comment:
Let's take the maximum invalid value.
In this case it's 17
```suggestion
    private static final User TOO_YOUNG = new User(17, "Bob", "passwd");
```